config.js

```diff
@@ -447,5 +447,7 @@ config.tincanapi = {
  */
 config.mixpanel = {
     'enabled': false,
+    'apikey': '',
+    'apisecret': '',
     'token': 'f3e9fce119d357b745a8dfa36248d632'
```

Comment:
Unfortunately these need to be kept here since the `mixpanel-data-export` module uses them to create the signature for the request.